docs/components/concepts/messages.md

```diff
@@ -1,9 +1,10 @@
 ---
 id: messages
 title: "Messages"
+description: "Learn how process instances can respond to incoming messages."
 ---
 
-Process instances can respond to incoming messages. Published messages must be mapped onto a process instance. This step is called **message correlation**.
+Process instances can respond to incoming messages. Published messages must be mapped onto a process instance. This step is called **[message correlation](/docs/guides/message-correlation.md)**.
```

Comment:
```suggestion
Process instances can respond to incoming messages. Published messages must be mapped onto a process instance. This step is called **[message correlation](/guides/message-correlation.md)**.
```

Reply:
I noticed `/docs` in a couple places in this PR. Is there something I'm missing here?

Reply:
@pepopowitz Can you assist? In which cases do we need to include /docs/ before routing to docs?

Reply:
This is a good catch by @akeller, and something I should tweak the automated URL-checking to catch, because it is definitely easy to get wrong. And tbh, I didn't realize this format was wrong until yesterday, when I ran into it while archiving 8.0. 

The `.md` at the end of the URL is important and good, but the `/docs/` at the beginning causes docusaurus to look in the `/docs/` source folder instead of the "current version" of the docs. So while `/docs/guides/message-correlation.md` links to the correct place in this version ("next"), it will not link correctly in other versions, as it will _always_ link to "next".

Hopefully that makes sense, but I had a hard time finding the words to explain it, so it's possible I failed.

So, Amara is correct -- we should likely never use `/docs/` for a link, even within the main docs instance, and even if we are also specifying the `.md` file extension. In this specific case, the path should be `/guides/message-correlation.md`.

Two things for me to do - (1) update the documentation guidelines to make this clear, and (2) update the link-checking to warn about this format of link.

Reply:
(the things for me to do are tracked in #2807.)

Reply:
Thank you @pepopowitz!! Would it be helpful for me to adjust this across all existing docs? I can launch an issue for it.

Reply:
> Would it be helpful for me to adjust this across all existing docs? I can launch an issue for it.

We definitely want to replace similar links at some point. I feel like there are probably an overwhelming amount to catch in one single issue, and IMO we should fix them as we see them. But if you feel like you've got the time to fix them all at once, go for it!